setup.py

```diff
@@ -298,6 +298,8 @@ def get_extensions():
     use_jpeg = use_jpeg and jpeg_found
     if use_jpeg:
         print("Building torchvision with JPEG image support")
+        print(f"  libpng include path: {jpeg_include}")
+        print(f"  libpng lib path: {jpeg_lib}")
```

Comment:
Small drive-by that helped me debug. We do the same for PNG already:

https://github.com/pytorch/vision/blob/0e62c34fb6ed7766fea368e67033564d8010ea78/setup.py#L278-L279

I wasn't able to get the version of the library in a way that would hold for all operating systems. I guess knowing the location is good enough.